### PR TITLE
Docs layout tweaks

### PR DESCRIPTION
**Pull request by simonbaird**
There are some more changes planned, but perhaps we can merge this is okay to merge as an iterative improvement.

Ref: https://issues.redhat.com/browse/EC-903

### CODE REVIEW

File: antora/supplemental-ui/partials/nav-explore.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{{!--
```
Comment (zregvart) [inline antora/supplemental-ui/partials/nav-explore.hbs:1]: We also want to set `--nav-panel-menu-height: 100%` in the `:root` of custom.css

**Review by zregvart — APPROVED** [review]
I'd set the titles directly in component's antora.yaml. One tweak to the CSS is needed to completely cover the place explorer bar took

File: antora/supplemental-ui/partials/nav-explore.hbs (hunk 1 above)
Comment (simonbaird) [reply antora/supplemental-ui/partials/nav-explore.hbs:1]: Good catch, thanks.

**Comment by simonbaird** [comment]
> I'd set the titles directly in component's antora.yaml.

I agree, but I want to do it later, since there's a reasonable chance they'll be changed again before I'm done. At that point I'm happy to remove the extra javascript/handlebars hackery.

**Pull request merged**